docs/README.md

```diff
@@ -117,6 +117,95 @@ Installation process
       helm install csi-baremetal csi/csi-baremetal-deployment --set driver.drivemgr.type=halmgr \
       --set global.registry=$REGISTRY --set global.registrySecret=$DOCKER_REGISTRY_SECRET
       ```
+Feature Supporting
+------
+### CIS hardening
+For supporting CIS Hardening we need to add the following settings while installing CSI and CSI Baremetal Operator:
+* While installing CSI Baremetal Operator set SecurityContext:
+  ```
+  --set securityContext.enable=true --set securityContext.runAsNonRoot=true --set securityContext.runAsUser=1000
```

Comment:
What does `1000` mean?

Reply:
It's just uid for container process (it's the same for the root namespace as well as the container namespace) and used in most cases when container process has no need in files modifications (as in our case). 
Generally there can be any uid here except 0 - in this case PSP admission controller will fail pod creation operation with error about root UID forbiddance. 
So, entrypoint will have also the same uid, as we state here, as owner respectively.

Reply:
Why do we need to ask user to set this parameter? Do we need to changes things like this in our docker files?
```
RUN addgroup -S bmcsi && adduser -S bmcsi -G bmcsi

USER bmcsi
```

Reply:
The user already exists. I described the problem above

Reply:
1. No need to user to choose user ID
2. Should be aligned with existing user in Docker files.